package.json

```diff
@@ -21,9 +21,13 @@
     "@mdx-js/react": "^1.6.22",
     "@smallstep/internal": "./vendor/internal",
     "@smallstep/step-ui": "./vendor/step-ui",
+    "@styled-icons/fa-solid": "^10.34.0",
```

Comment:
Is this still needed? We should use all material icons, so could we uninstall this dependency?

Reply:
None of the styled stuff should be needed, we can get rid of that dependency!